app/javascript/crayons/MarkdownToolbar/icons.jsx

```diff
@@ -0,0 +1,211 @@
+import { h } from 'preact';
+
+export const Bold = () => (
```

Comment:
The problem with that is we can't use the SVG assets in JSX, unless we used an `<img>` tag and used the asset as the src... that's why we have duplication of SVGs peppered throughout the Preact codebase 🙁

Reply:
I had a look into packages like this a bit before and at the time, none of them seemed to meet our needs, particularly with the way we use `aria-labelledby` and SVG title IDs to label some icon buttons... In saying that though, I've since learned that labelling icons with SVG titles isn't super well supported across assistive technologies, so maybe it would be good to revisit.

It's absolutely a pain point having SVG code duplicated everywhere & would be good to solve that